app/models/name/format.rb

```diff
@@ -96,16 +96,18 @@ def self.group_suffix(name)
   # Make sure display names are in boldface for accepted names, and not in
   # boldface for deprecated names.
   def self.make_sure_names_are_bolded_correctly
-    Name.connection.select_values(%(
-      SELECT id FROM names
-      WHERE IF(deprecated, display_name LIKE "%*%", display_name NOT LIKE "%*%")
-    )).map do |id|
-      name = Name.find(id)
+    msgs = ""
+    needs_fixing = Name.where(deprecated: true).
+                   where(Name[:display_name].matches("%*%")).
+                   or(Name.where(deprecated: false).
+                      where(Name[:display_name].does_not_match("%*%")))
+    needs_fixing.to_a.each do |name|
       name.change_deprecated(name.deprecated)
       name.save
-      "The name #{name.search_name.inspect} " \
-      "should #{name.deprecated && "not "} have been in boldface."
+      msgs += "The name #{name.search_name.inspect} " \
+              "should #{name.deprecated && "not "} have been in boldface."
```

Comment:
Good catch. But to prevent a potential double space when the "not" is *not* added (I believe this is a possibility), i'll remove the space outside the bracket instead.